qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferDbConnector.groovy

```diff
@@ -147,6 +151,14 @@ class OfferDbConnector implements CreateOfferDataSource{
         return offerOverviewList
     }
 
+    static OfferId parseOfferId(String offerId) {
+        def splitId = offerId.split("_")
+        def projectPart = splitId[0]
+        def randomPart = splitId[1]
+        def version = splitId[2]
+        return new OfferId(projectPart, randomPart, version)
+    }
+
```

Comment:
same for this. I think this is logic that should be part of the ID-Converter to move out the logic of the DB part and also to test this behavior properly (Humble Objects)

Reply:
please see my above comment